### PR TITLE
[BottomNavigation] Extend ripple to bar bounds.

### PR DESCRIPTION
**Pull request by romoore**
The ripple effect was being constrained to the layout container of the
BottomNavigationBar. Instead, it should be allowed to fill the visible
bar. It also correct a bug where the ripple effect would extend beyond
the bounds of the bar because the item views were taller than the bar
view.

The effect was most prominent on newer iPhones where `safeAreaInsets` pushes the item views away from the edge of the screen.

## iPhone XS Max (Simulator)

|Before|After|
|---|---|
|bn-ripple-ipx-develop|bn-ripple-ipx-fixed|

### CODE REVIEW

File: components/BottomNavigation/src/MDCBottomNavigationBar.m [hunk 1]
```diff
@@ -101,17 +102,22 @@ - (void)commonMDCBottomNavigationBarInit {
...
+  _itemsLayoutView.clipsToBounds = NO;
+  [self addSubview:_inkClippingView];
+  [_inkClippingView addSubview:_itemsLayoutView];
 #if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunguarded-availability"
 #pragma clang diagnostic ignored "-Wtautological-pointer-compare"
   if (&UIAccessibilityTraitTabBar != NULL) {
-    _containerView.accessibilityTraits = UIAccessibilityTraitTabBar;
+    _itemsLayoutView.accessibilityTraits = UIAccessibilityTraitTabBar;
```
Comment (codeman7) [inline components/BottomNavigation/src/MDCBottomNavigationBar.m:120]: _Tip:_ Should we mark the `inkClippingView` as `accessibilityElement = NO`?

File: components/BottomNavigation/src/MDCBottomNavigationBar.m [hunk 2]
```diff
@@ -164,27 +172,27 @@ - (void)layoutLandscapeModeWithBottomNavSize:(CGSize)bottomNavSize
...
+      [self sizeitemsLayoutViewItemsDistributed:NO
                             withBottomNavSize:bottomNavSize
                                containerWidth:containerWidth];
       self.titleBelowItem = YES;
       break;
   }
 }
 
-- (void)sizeContainerViewItemsDistributed:(BOOL)itemsDistributed
+- (void)sizeitemsLayoutViewItemsDistributed:(BOOL)itemsDistributed
```
Comment (codeman7) [inline components/BottomNavigation/src/MDCBottomNavigationBar.m]: _Tip:_ Shouldn't it be `sizeItemsLayoutViewItemsDistributed..`? `sizeItems` ~`sizeitems`~?

**Review by codeman7 — APPROVED** [review]

File: components/BottomNavigation/src/MDCBottomNavigationBar.m (hunk 1 above)
Comment (romoore) [reply components/BottomNavigation/src/MDCBottomNavigationBar.m:120]: For UIView, the default value is `NO`. 

From UIAccessibility.h:

```objc
/*
 Return YES if the receiver should be exposed as an accessibility element. 
 default == NO
 default on UIKit controls == YES 
 Setting the property to YES will cause the receiver to be visible to assistive applications. 
 */
@property (nonatomic) BOOL isAccessibilityElement;
```

File: components/BottomNavigation/src/MDCBottomNavigationBar.m (hunk 2 above)
Comment (romoore) [reply components/BottomNavigation/src/MDCBottomNavigationBar.m]: Good catch - that was a find/replace mistake.

**Pull request merged**